pkgs/by-name/pi/picom-pijulius/package.nix

```diff
@@ -0,0 +1,19 @@
+{ picom, lib, fetchFromGitHub }:
+picom.overrideAttrs (oldAttrs: rec {
+  pname = "picom-pijulius";
+  version = "e7b1488";
```

Comment:
The git commit should not be the version please follow the version sheme for unstable versions from https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#conventions

Reply:
Will do